src/main/java/uk/gov/hmcts/reform/managecase/repository/CaseTypeDefinitionVersion.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.managecase.repository;
+
+import java.io.Serializable;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+@Getter
+@Setter
+public class CaseTypeDefinitionVersion implements Serializable {
+
```

Comment:
May be simple @Data annotation?  That would give all other required ones including toString()